### ISSUE TITLE
The New Editor style is bad in the Documention description editor

### ISSUE THREAD

**Issue by hlmd** [a85/POSTMan-Chrome-Extension#10417]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

The New Editor style is bad in the Documention description editor.

### Steps To Reproduce

1. Switch to Scratch Pad
2. Open an existing Collection|folder|request
3. Open the Documentation on the right sidebar
4. Clieck the Edit icon to Edit description
5. Use the New editor

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/45023268/140038760-573fd170-cae9-4576-ae66-d71550f92d18.png)

### Environment Information

```markdown
- Operating System: Windows 10 21H1
- Platform Type: Native App
- Postman Version: 9.1.3
```

**Comment by Krishks369** [comment]
I would like to work on this issue but I don't have much clarity on the issue. please say what is to be done.
Should the docs be updated or bug has to be fixed.
if yes please guide how to start

**Comment by hlmd** [comment]
This issue has been resolved in version 9.1.4.
That's cool.1
